Mysql error when new player connected

```
(client) RedEM: Player activated: Pablo
[31m[ERROR] [MySQL] [redem_roleplay] An error happens on MySQL for query "INSER
T INTO characters (`identifier`, `firstname`, `lastname`, `characterid`) VALUES
('steam:110000107a1417a', 'pablo', 'escobaro', 1);": ER_NO_DEFAULT_FOR_FIELD: Fi
eld 'id' doesn't have a default value[0m
Character made!
That character does not exist!
SCRIPT ERROR: @redemrp_skin/server/sv_main.lua:6: attempt to index a nil value (
local 'user')
> ref (@redemrp_skin/server/sv_main.lua:6)
> handler (@redem_roleplay/server/sv_main.lua:83)
> handler (@redemrp_skin/server/sv_main.lua:5)
```

same here
[ERROR] [MySQL] [redem_roleplay] An error happens on MySQL for query "INSERT INTO characters (`identifier`, `firstname`, `lastname`, `characterid`) VALUES ('steam:"id..."', 'Benjamin', 'Flowers', 1);": ER_NO_DEFAULT_FOR_FIELD: Field 'id' doesn't have a default value

--- a/db.sql
+++ b/db.sql
@@ -29,7 +29,7 @@ CREATE TABLE IF NOT EXISTS `characters` (
   `firstname` varchar(50) COLLATE utf8mb4_bin DEFAULT 'first',
   `lastname` varchar(50) COLLATE utf8mb4_bin DEFAULT 'last',
   `jobgrade` int(11) DEFAULT 0,
-  `id` int(11) NOT NULL
+  `id` int(11) NOT NULL AUTO_INCREMENT
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;
 
 -- Data exporting was unselected.

